### PR TITLE
feat: fix openapi generating

### PR DESCRIPTION
**Pull request by xburda4**
Fix openapi generating and add a test to check the content of the generated file

### CODE REVIEW

File: cmd/tea/oapi_compose.go [hunk 1]
```diff
@@ -6,13 +6,11 @@ import (
...
 
-	cmderrors "go.strv.io/tea/pkg/errors"
-	"go.strv.io/tea/pkg/openapi/load"
-
 	"github.com/go-openapi/spec"
-	"github.com/go-openapi/swag"
 	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v2"
+	cmderrors "go.strv.io/tea/pkg/errors"
+	"go.strv.io/tea/pkg/openapi/load"
```
Comment (Fazt01) [inline cmd/tea/oapi_compose.go:12]: keep imports from project and external imports separate

File: go.mod [hunk 2]
```diff
@@ -13,6 +13,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/inconshreveable/mousetrap v1.0.1 // indirect
```
Comment (Fazt01) [inline go.mod:16]: probably should be together with other indirect dependencies?

File: pkg/openapi/load/spec.go [hunk 3]
```diff
@@ -6,10 +6,9 @@ import (
 	"encoding/json"
 	"fmt"
 
-	"go.strv.io/tea/pkg/openapi/spec"
-
 	oapi "github.com/go-openapi/spec"
 	"github.com/go-openapi/swag"
+	"go.strv.io/tea/pkg/openapi/spec"
```
Comment (Fazt01) [inline pkg/openapi/load/spec.go:11]: keep separate

File: cmd/tea/oapi_compose_test.go [hunk 4]
```diff
@@ -56,7 +54,26 @@ func Test_runOAPICompose(t *testing.T) {
...
+			defer func() {
+				if cleanupAfterTest {
+					require.NoError(t, os.Remove(outputFilePath))
+				}
+			}()
 			assert.NoError(t, tt.cond(t))
+
+			tofp, err := os.ReadFile(tt.args.testOutputFilePath)
+			if err != nil {
+				t.Errorf("runOAPICompose() error = %v, wantErr %v", err, tt.wantErr)
```
Comment (Fazt01) [inline cmd/tea/oapi_compose_test.go:66]: expected output file reading should never fail (misleading error message, its not a result of `runOAPICompose` at all)

File: cmd/tea/oapi_compose_test.go [hunk 5]
```diff
@@ -56,7 +54,26 @@ func Test_runOAPICompose(t *testing.T) {
...
+				return
+			}
+
+			ofp, err := os.ReadFile(tt.args.opts.OutputFilePath)
+			if err != nil {
+				t.Errorf("runOAPICompose() error = %v, wantErr %v", err, tt.wantErr)
```
Comment (Fazt01) [inline cmd/tea/oapi_compose_test.go:72]: this one could fail, but the message should reflect that (its not an error of `runOAPICompose` directly, but a missing result file)

**Review by TomasKocman — APPROVED** [review]

**Review by Fazt01 — APPROVED** [review]

**Pull request merged**